Nev create rich readme (#140)
UKHO/docs: README.md

# Software Engineering at the UKHO

> This repo contains documentation about digital delivery teams, behaviors and processes at the UKHO.

Reviewer: ```suggestion
This repository contains documentation about digital delivery teams, their behaviours and processes and our engineers at the UKHO.
```

> We have eight software engineering teams that develop and support software using the .Net and Java stacks.  

Reviewer: Is this limiting - what about JavaScript, the tools used in DE/DS ?

Author: Yep, will ammend.

We’re proud that we’ve created this group by recruiting people who share a commitment to professional development and watched them grow with us.   Right now, we have people pursuing certification in Azure, AWS, Kubernetes and Security; we have others on management training courses; we even have people on Masters Degree programmes. 

> 

Reviewer: Two blank lines 🙄

Author: ok

## How we work

> Everything we do is team based.  We take Agile seriously; your delivery team will be small but will have the people it needs to get things done - developers, testers, product owner and a delivery manager - as well as access to infrastructure specialists, UX experts and analysts. 

Reviewer: ```suggestion
Everything we do is team based.  We take Agile seriously; your delivery team will be small but will have the people it needs to get things done - developers, testers, a product owner and a delivery manager - as well as access to infrastructure specialists, UX experts and analysts. 
```

> Quality is paramount.  Our [engineering](./docs/software-engineering-policies/README.md) and [quality assurance](./docs/quality-assurance/README.md) policies support our teams to do good work.  Most code is done [in pairs](./docs/software-engineering-policies/PairProgramming/PairProgrammingPolicy.md) or larger groups to ensure quality and spread knowledge.  Everyone [reviews code](./docs/software-engineering-policies/CodeReview/CodeReviewPolicy.md) and everyone welcomes feedback. 

Reviewer: ```suggestion
Quality is paramount.  Our [engineering](./docs/software-engineering-policies/README.md) and [quality assurance](./docs/quality-assurance/README.md) policies support our teams to do good work.  Most code is done [in pairs](./docs/software-engineering-policies/PairProgramming/PairProgrammingPolicy.md) or larger groups to ensure quality and to spread knowledge.  Everyone [reviews code](./docs/software-engineering-policies/CodeReview/CodeReviewPolicy.md) and everyone welcomes feedback. 
```

We strive to automate as much as possible: [testing](./docs/quality-assurance/test-strategy.md), builds, deployments use the latest automation tools available.

> We actively try and pay down [technical debt](./docs/software-engineering-policies/TechnicalDebt/TechnicalDebtGuidance.md).

Reviewer: ```suggestion
We actively look to pay down [technical debt](./docs/software-engineering-policies/TechnicalDebt/TechnicalDebtGuidance.md).
```

> ## Our community

Reviewer: We use the terms "community", "Practice" and "Department" in the first two paragraphs - can we agree a common term to use.

Author: OK, fixed

Within this practice, we encourage the formation of other communities to support our interests and our work.

The community of Lead Developers provides technical leadership for the department, recommending tools, technologies and techniques for adoption.  This group contains people with various specialisms and levels of experience who all support each other in their roles.

Our [Security Champion](./docs/software-engineering-policies/SecureDevelopment/SecurityChampion.md) community is made up of people interested in [application security](./docs/software-engineering-policies/SecureDevelopment/SecureDevelopmentPolicy.md), and coaches teams to create safer software, as well as increasing their own skills and qualifications.

We also have many other communities of interest covering such things as Accessibility, Testing, UX, Azure, AWS and .Net.

## Balance

> We offer a great work / life balance.  Peoples’ start and end times are built around their own preferences and around a 10am – 2pm team time, where the team is focussed on building things (no meetings!).  We’re all set up for working from home and encourage people to make use of this if they prefer.  We are committed to providing part time options and reasonable accommodations to help people meet their needs. 

Reviewer: ```suggestion
We offer a great work / life balance - start and end times are fluid, centred around a 10am – 2pm team time, where the team is focussed on building things (no meetings!).  We’re all set up for working from home and encourage people to make use of this if they prefer.  We are committed to providing part-time options and offer reasonable accommodations to help people meet their needs. 
```

> You can find all our documentation at https://docs.ukho.dev

Reviewer: Needed? To be reading this page they must be at that URL or the GH repo already.

Author: will remove
